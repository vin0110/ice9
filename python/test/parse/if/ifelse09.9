# fails
if 8 -> 9;
[] else -> 2;
[] else -> 4;
fi
