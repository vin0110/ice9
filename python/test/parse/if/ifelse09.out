ParseError: line 4: unexpected token; expected FI, got BOX
